### ISSUE TITLE
 Fix up mdev.conf [trac #833]

### ISSUE THREAD

**Issue by bradfa** [cross-lfs/clfs-embedded#9]
Our mdev.conf may not be the best example of how to do an mdev.conf. Based on this mailing list thread (​http://lists.busybox.net/pipermail/busybox/2009-November/070889.html), the busybox people feel we may have an older version of an example mdev.conf that could not be exploiting all the features of mdev.

Take a look at if mdev.conf should be updated to better fit with current busybox ideals.

The current mdev.conf works for me, so this isn't a huge priority.

This is trac ticket: https://trac.clfs.org/ticket/833

**Comment by bradfa** [comment]
Alpine Linux uses Busybox and has an example mdev.conf that could be useful to base ours off of: ​http://git.alpinelinux.org/cgit/aports.git/plain/main/busybox-initscripts/mdev.conf?h=2.1-stable

May also need to revert commit 08718a41c516aff0a0e02a8267b0190b456f0d9d that removed the mknod'ing of /dev/null and /dev/console. My beagleboard-xm doesn't appear to need those nodes created before starting mdev but sjs205's board does or else there's no output to the serial console, only the video console.

**Comment by bradfa** [comment]
In my repo (​http://git.cross-lfs.org/?p=abradford/clfs-embedded.git), three commits update mdev.conf and revert the commit that removed the mknod'ing of /dev/console and /dev/null.

Commit numbers are: 111f2c62a7d0fd409c12417aedc7313d0b95deac, 4d261af39d6ef04839271f30ef97a227b4351c0d, 0e05c5c71025f637f18989a5452e3a1e198cca7b

If these are accepted into the mainline embedded book, this ticket can be closed. Further improvements to mdev.conf should be another ticket (if needed).
